cli/crates/cli/src/output/report.rs

```diff
@@ -55,13 +62,17 @@ pub fn project_created(name: Option<&str>) {
     } else {
         watercolor::output!(r#"✨ Your project was successfully set up for Grafbase!"#, @BrightBlue);
 
-        let schema_path = &[".", GRAFBASE_DIRECTORY_NAME, GRAFBASE_SCHEMA_FILE_NAME].join(&slash);
+        let schema_path = &[".", GRAFBASE_DIRECTORY_NAME, schema_file_name].join(&slash);
 
         println!(
             "Your new schema can be found at {}",
             watercolor!("{schema_path}", @BrightBlue)
         );
     }
+
+    if let ProjectTemplate::FromDefault(ConfigType::TypeScript) = template {
+        println!("Install the added dependencies by running your project's package manager.");
```

Comment:
We could make it a bit friendlier maybe

Something like

"We've added our SDK to your package.json, make sure to install dependencies before continuing"

maybe package.json could be blue, not sure.